collection.c

```diff
@@ -1622,7 +1631,11 @@ static void php_mongocollection_remove(zval *this_ptr, mongo_collection *c, zval
 		RETURN_FALSE;
 	}
 
-	if (php_mongo_api_connection_supports_feature(connection, PHP_MONGO_API_WRITE_API)) {
+	is_acknowledged = is_gle_op(this_ptr, z_write_options, &link->servers->options TSRMLS_CC);
+	supports_command = php_mongo_api_connection_supports_feature(connection, PHP_MONGO_API_WRITE_API);
+	supports_opcode = php_mongo_api_connection_supports_feature(connection, PHP_MONGO_API_RELEASE_2_4_AND_BEFORE);
```

Comment:
I'm not too inclined to create a new function, as we already have a bunch of static helpers in this file. I'm open to a macro for the conditional expression or adding a block comment above each of the three conditionals to explain the reasoning.